.github/workflows/ci.yml

```diff
@@ -570,6 +590,16 @@ jobs:
           SS_BASE_URL="http://localhost"
           EOF
 
+          if [[ $GITHUB_REPOSITORY =~ /(silverstripe-dynamodb)$ ]]; then
+            cat << EOF >> .env
+          AWS_DYNAMODB_ENDPOINT="http://localhost:8000"
```

Comment:
Should this stuff be indented to match the indentation of `cat`?

Reply:
No, it's deliberate, this indentation is correct. We do the same thing higher up in the file.